test/e2e/framework/node/cpuset.go

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Why don't we implement this package under ` test/e2e_node/`?
Those functions seem to be used in ` test/e2e_node/` only.

Reply:
Thanks for raising this. I can move back under `test/e2e_node`, no problem.
In general is worth noting that this package actually duplicates kubelet code (https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet/cm/cpuset), this is because AFAIU we should not use kubelet production code in the test implementation itself, so a little duplication is better. Is my understanding right?

Reply:
We have moved test specific functions from e2e test framework (test/e2e/framework/) already with much effort as https://github.com/kubernetes/kubernetes/issues/84380 I'd like to avoid such work again.
e2e test framework should not depend on the other packages in kubernetes/kubernetes repo.
Your understanding seems good for me, but if we move those functions from the framework, we don't need to make duplication because of the outside of the framework.

Reply:
Thanks for your feedback. I'll updated a much narrower PR soon. I still believe there is some merit reorganizing this code under a subpackage in `test/e2e_node`. But no more additions to the framework, and no more duplicated packages.